src/azure_hl.rs

```diff
@@ -1781,6 +1793,13 @@ impl AzPoint {
             y: point.y,
         }
     }
+
+    fn from_vector_2d(point: &Vector2D<AzFloat>) -> Self {
```

Comment:
No idea, I really just followed whatever was already in place (but asked myself the same question), there's also something very similar in the servo repo in the code that interacts with azure (something that should just use From/Into).